docs/platform/release-notes/tizen-7-0-m1.md

```diff
@@ -0,0 +1,545 @@
+
+
+### Lightweight Web Solution
+
+#### New and changed features
+
+- Lightweight Web Engine
+  - Web Engine
+    - Dropdown menu feature has been added.
+    - CSS Custom property has been added.
+    - CSS mask-image property has been added.
+    - Rendering performance for css custom property has been optimized.
+    - Painting performance for gradient has been improved.
+  - Javascript Engine
+    - WeakRef and finalizationRegistry specification have been added.
+    - JS debugger has been added.
+- JS based lightweight backend service FW
+  - Lightweight node.js
+    - Customizable exception handling has been added.
+    - Serializer and deserializer have been added.
+    - IndexedPropertyHandlers have been added.
+
+#### Fixes
+
+- Lightweight Web Engine
+  - WebSocket memory leak defects have been fixed.
+  - Grid layout defects have been fixed.
+- Lightweight node.js
+  - Promise defects have been fixed.
+  - Template defects have been fixed.
+  - Multithread defects have been fixed.
+  - StackTrace defects have been fixed.
+  - V8 modules have been hidden by default.
+
+
+### Tizen .NET
+
+#### New and changed features
+
+- .NET
+  - TizenFx
+    - Some Slider events have been removed from Tizen.NUI.
+    - Sticker class has been added to Tizen.Uix.
+    - RepeatedText class has been added to Tizen.Uix.Tts.
+    - New module errors have been added to Tizen.Uix.Tts.
+    - MediaFileSource class has been added to Tizen.Multimedia.Remoting.
+    - BufferedAmountLow event has added to Tizen.Multimedia.Remoting.
+    - GLView has been added to NUI.
+    - PrimaryCursorPosition has been added to TextField and TextEditor of NUI.
+    - InputHint has been added to Tizen.Uix.InputMethod.
+  - .NET runtime and tools
+    - .NET Runtime has been upgraded from NETCore 3.1.3 to .NET 6.
+
+
+### Toolchain
+
+#### New and changed features
+
+- Hardware-assisted AddressSanitizer (HWASan)
+  - GCC/ Libsanitizer : HWASan feature has been backported and tuned for Tizen.
+  - HWASan compiler options have been applied.
+  - HWASan runtime options have been applied to build and runtime environment.
+  - Support for HWASan has been added to glibc and packages.
+- Large File Support (LFS)
+  - LFS support has been added to AArch64.
+- New architecture(ARMv7hl) support
+  - Build support has been added for 32-bit hard float ABI.
+
+#### Fixes
+
+- Binutils/ LTO bug #25355
+  - LTO plugin bugfix has been backported into binutils, ld, and bfd.
+- Glibc/ dynamic linker bug #24259
+  - Dynamic linker crash after a previously failed call to dlopen has been fixed.
+- Toolchain testsuites bugs have been fixed on GCC, Glibc and Binutils.
+
+
+### Machine Learning
+
+#### New and changed features
+
+- Machine Learning (ML) Inference API updates
+  - capi-machine-learning-inference-single package have been added so that devices may have SingleShot API without installing Pipeline API, GStreamer, and NNStreamer's core packages.

```

Comment:
```suggestion
  - Capi-machine-learning-inference-single package has been added so that devices may have SingleShot API without installing Pipeline API, GStreamer, and NNStreamer's core packages.

```

Reply:
'capi-machine-learning-inference-single' is a package name.